### DISCUSSION TITLE
Document uninstallation procedure

### DISCUSSION THREAD

**Discussion by vsandvold** [rxhanson/Rectangle#554] [Q&A]
Hi! Thanks for taking time to develop such a solid product like Rectangle. Unfortunately, I don't have a use for it after all. Since I'm not an expert Mac use, I'm unsure about how to properly uninstall Rectangle. Especially since it starts up on boot, etc. Is it as easy as just dragging the app from Programs into the trash, or do I have to hunt down additional configuration files? Does the app register as some kind of service that I must disable? I would really appreciate if you can add some basic uninstallation instructions to Github or the product homepage. Thanks in advance!

**Comment by rxhanson** [comment]
https://github.com/rxhanson/Rectangle#uninstallation
